chef/cookbooks/database/recipes/crowbar.rb

```diff
@@ -45,7 +45,7 @@ def pg_hash_only_merge(merge_onto, merge_with)
 # override) as normal attributes, so fix it here
 # Note that the postgresql.client key should never be in normal, so this means
 # we'll do that only once.
-if node.normal_attrs['postgresql'].has_key? 'client'
+if !node.normal_attrs['postgresql'].nil? && node.normal_attrs['postgresql'].has_key?('client')
```

Comment:
```if node.normal_attrs['postgresql'] && node.normal_attrs['postgresql'].has_key? 'client'```
That should be enough, and stop the braces, we are writing ruby and not python :)